packages/shared/src/hooks/vote/useFeedVotePost.ts

```diff
@@ -43,14 +43,6 @@ export const useFeedVotePost = ({
 
   const { toggleUpvote, toggleDownvote, ...restVotePost } = useVotePost({
     variables: { feedName },
-    onMutate: ({ id, vote }) => {
-      return mutateVoteFeedPost({
-        id,
-        vote,
-        items,
-        updatePost,
-      });
-    },
   });
```

Comment:
@capJavert Because of set message above we actually don't need this here, we can just rely on the mutationSubscription for the feed ones

Reply:
One thing that this changes though is that upvote will not show optimistically until server side call actually passes as to on current production upvote will instantly turn green on click.

Reply:
Hmm that's true, not sure if it's acceptable or if you have any other ideas on how we can promote it to both mutations optimistically?

Reply:
This brings back the initial discussion around actually wanting to do both updates in the same onMutate ideally.
Else one will always be after the mutation happens.